grails-app/i18n/messages.properties

```diff
@@ -3414,7 +3414,7 @@ react.locationsConfiguration.skipConfiguration.label=Skip Configuration for now
 react.locationsConfiguration.additionalTitle.label=Fill in the details for your location. Click the question mark next to the field name to find out more.
 react.locationsConfiguration.details.label=Details
 react.locationsConfiguration.locationNumber.label=Location Number
-react.locationsConfiguration.locationNumber.tooltip.label=Enter a short code to identify the location. Often users choose an acronym, like “NDH” for “Neno District Hospital”.
+react.locationsConfiguration.locationNumber.tooltip.label=Enter a short code to identify the location. Often users choose an acronym, like “NDH” for “Neno District Hospital” <a href='https://openboxes.atlassian.net/wiki/spaces/OBW/pages/1291452471/Configure+Organizations+and+Locations'>Link to Something</a>.
```

Comment:
Actually, links should not be specified here. I think we'd need to do it in a configurable way. In the config should be a property with map of links that we will be able to pull through an `/api/supportLinks` endpoint. For example:
```
openboxes.supportLinks = [
    locationName: 'https://openboxes.atlassian.net/wiki/spaces/OBW/pages/1291452471/Configure+Organizations+and+Locations',
]
```
and then on the front end side the tooltip content should be built in the `FIELDS` declaration based on a translated message with link and passed down to `Select` with attributes.